ra-front/src/app/services/store.service.ts

```diff
@@ -20,6 +20,9 @@ export class StoreService {
 //   public baseURL = 'http://localhost:3333';
   // public baseURL = 'http://localhost:8080';
   public baseURL = 'http://larissa-ra-back.azurewebsites.net';
+    
+  // public barrierURL = 'http://raptors-barrier-generator.herokuapp.com/'
+  public barrierURL = 'http://localhost:5000/'
```

Comment:
Running our project locally will start it on port 5000 if not specified otherwise.